MdePkg/Library/DxeRngLib/DxeRngLib.c

```diff
@@ -32,7 +32,7 @@ typedef struct {
 // These represent UEFI SPEC defined algorithms that should be supported by
 // the RNG protocol and are generally considered secure.
 //
-GLOBAL_REMOVE_IF_UNREFERENCED SECURE_RNG_ALGO_ARRAY  mSecureHashAlgorithms[] = {
+STATIC GLOBAL_REMOVE_IF_UNREFERENCED SECURE_RNG_ALGO_ARRAY  mSecureHashAlgorithms[] = {

```

Comment:
Use `static` for this and an improvement is to prefix the name of the global with the module/lib base name. 

`mDxeRngLibSecureHashAlgorithms`

Reply:
I'd prefer to keep the existing short names.  The variables are used within the source file only, so `static` alone is enough to fix the name clash.

Reply:
I agree this resolves the issue.  The use of the lib name prefix is required for non-static globals.